src/Wpf.Ui/Controls/NavigationView/NavigationView.Properties.cs

```diff
@@ -15,6 +15,22 @@ namespace Wpf.Ui.Controls;
 
 public partial class NavigationView
 {
+    /// <inheritdoc/>
```

Comment:
This `inheritdoc` does not lead to anything, it has no documentation in `INavigationView`. Additionally, it has no equivalent in WPF or Win UI. Which makes the name quite unintuitive.